src/interpreter.ts

```diff
@@ -705,8 +707,54 @@ function run(evaluation: Evaluation, natives: Natives, sentences: List<Sentence>
   return currentFrame.operandStack.length ? evaluation.instances[currentFrame.popOperand()] : null
 }
 
+interface TestResult {
+  error?: Error,
+  duration: number,
+  evaluation: Evaluation,
+}
+
+function runTest(evaluation: Evaluation, natives: Natives, test: Test): TestResult {
+  log.resetStep()
+
+  if (test.parent().is('Describe')) {
+    const describe = test.parent() as Describe
+    const describeInstanceId = evaluation.createInstance(describe.fullyQualifiedName())
+
+    evaluation.pushFrame(compile(evaluation.environment)(
+      ...describe.variables(),
+      ...describe.fixtures().flatMap(fixture => fixture.body.sentences),
+    ), describeInstanceId)
+
+    try {
+      stepAll(natives)(evaluation)
+    } catch (error) {
+      return {
+        error,
+        duration: 0,
+        evaluation,
+      }
+    }
+  }
+
+  let error: Error | undefined
+
+  const before = process.hrtime()
+  try {
+    run(evaluation, natives, test.body.sentences)
+  } catch (e) {
+    error = e
+  }
+  const delta = process.hrtime(before)
+
+  return {
+    evaluation,
+    error,
+    duration: round(delta[0] * 1e3 + delta[1] / 1e6),
```

Comment:
I won't say anything about this magic calculus...

Reply:
These monsters return a touple with seconds and nanoseconds. If you can think of a clearer way to translate it to ms I'm all ears

Reply:
I wonder why people with low-design skills take cards related with dates and times in core implementations (like java.util.Date, java.sql.Date, and also in JS).

I feel this is not the right moment, but maybe sometime we can use [momentjs simpler approach](https://momentjscom.readthedocs.io/en/latest/moment/04-displaying/07-difference/).

Reply:
I'm not complaining about your solution, just for the record. I believe having a tuple with seconds and nanoseconds is obscure and forces you to remember that ugly interface for curious people like me.

Reply:
Nah, I know. I´m not fond of it either...
Maybe there is some other interface somewhere to measure time in ms? I would rather not add a dependency just for this, though...